frontend/src/views/IntegrationsPage/components/IntegrationsSection/components/IntegrationDetails.tsx

```diff
@@ -52,7 +52,8 @@ export const IntegrationDetails = ({ integration }: Props) => {
           <FormLabel
             label={
               (integration.integration === "qovery" && integration?.scope) ||
-              (integration.integration === "circleci" && "Project") ||
+              (integration.integration === "circleci" &&
+                (integration.scope === "context" ? "Context" : "Project")) ||
```

Comment:
Use enum here. You could even just print the enum value, with the first letter capitalized, and avoid the ternary operator all together.

Reply:
yeah will use enum value here but note that we can't just capitalize first letter because we're not doing any backfilling - existing circle ci integrations won't have the scope value